src/test/matchers/golden.ts

```diff
@@ -101,6 +105,7 @@ export function compare(
     if (isWriteMissingMode) {
       fs.mkdirSync(path.dirname(snapshotFile), { recursive: true });
       fs.writeFileSync(snapshotFile, actual);
+      fs.writeFileSync(actualPath, actual);
```

Comment:
Let's add the actualPath to the `writing actual` message below so that I know where to look at?

Reply:
Discussed offline and decided to keep as-is.